src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -15,22 +15,24 @@
 #include "winrt/Windows.System.UserProfile.h"
 #include "../../types/inc/Utils.hpp"
 
+#include "winrt/Windows.Web.Http.Filters.h"
+
 using namespace ::Microsoft::Console;
 using namespace ::Microsoft::Terminal::Azure;
-
-using namespace web;
-using namespace web::http;
-using namespace web::http::client;
-using namespace web::websockets::client;
 using namespace winrt::Windows::Security::Credentials;
 
-static constexpr int CurrentCredentialVersion = 1;
-static constexpr auto PasswordVaultResourceName = L"Terminal";
-static constexpr auto HttpUserAgent = L"Terminal/0.0";
+static constexpr int CurrentCredentialVersion = 2;
+static constexpr std::wstring_view PasswordVaultResourceName = L"Terminal";
+static constexpr std::wstring_view HttpUserAgent = L"Mozilla/5.0 (Windows NT 10.0) Terminal/1.0";

```

Comment:
![image](https://user-images.githubusercontent.com/2256941/216719995-dca00cb0-cce0-43ca-a66d-a6fd1e4eefa2.png)

Reply:
YOU'RE NOT WRONG

Reply:
Heh. If there is, it almost certainly requires us to move off of device code based authentication... which is work I've got booked for "some time after March".